settings.sh.in

```diff
@@ -70,38 +70,34 @@ PGADMIN_KRB5_DIR="C:\\\\Program Files\\\\MIT\\\\Kerberos"
 YARN_HOME="C:\\\\pgBuild64\\\\Yarn"
 NODEJS_HOME="C:\\\\pgBuild64\\\\node-v18.18.2-win-x64"
 
-PG_SSH_OSX=buildfarm@battersea-park.ox.uk.enterprisedb.com
+PG_SSH_OSX=buildfarm@macbuildfarm-prod-cyt-3.edbcorp.net
 PG_PATH_OSX=/opt/pginstaller_17.auto
 PG_SSH_OSX_SIGN=buildfarm@cadwell.ox.uk.enterprisedb.com
 PG_PATH_OSX_SIGN=/Users/buildfarm/pginstaller_17.auto
 PG_PATH_OSX_SIGNTOOL=/opt/local/bin/osxsigner
 PG_SSH_OSX_NOTARY=buildfarm@cadwell.ox.uk.enterprisedb.com
 PG_PATH_OSX_NOTARY=/Users/buildfarm/pginstaller_17.notary
 PG_PGHOME_OSX=$PG_PATH_OSX/server/staging_cache/osx
-PG_MAVEN_HOME_OSX=/opt/local/Current/apache-maven-3.8.2
-PG_DOCBOOK_OSX=/usr/local/docbook-xsl-1.77.1
 PG_CMAKE_OSX=/Applications/CMake.app/Contents/bin/cmake
-PG_PERL_OSX=/Library/edb/languagepack/v4/Perl-5.38
-PG_PYTHON_OSX=/Library/edb/languagepack/v4/Python-3.11
+PG_PERL_OSX=/Library/edb/languagepack/v5/Perl-5.40
+PG_PYTHON_OSX=/Library/edb/languagepack/v5/Python-3.12
 PGADMIN_PYTHON_OSX=/Library/Frameworks/Python.framework/Versions/Current
-PG_TCL_OSX=/Library/edb/languagepack/v4/Tcl-8.6
-PG_LANGUAGEPACK_OSX=/Library/edb/languagepack/v4
+PG_TCL_OSX=/Library/edb/languagepack/v5/Tcl-8.6
+PG_LANGUAGEPACK_OSX=/Library/edb/languagepack/v5
 PGSERVER_STAGING_OSX=$PWD/server/staging/osx/server
 PGADMIN_STAGING_OSX=$PWD/server/staging/osx/pgadmin4
 SB_STAGING_OSX=$PWD/server/staging/osx/stackbuilder
 CLT_STAGING_OSX=$PWD/server/staging/osx/commandlinetools
 PG_INSTALLER_DEFAULT_PATH=/Library/PostgreSQL/17
-YARN_HOME_OSX=/opt/local/Current_v15/yarn
-NODEJS_HOME_OSX=/opt/local/Current_v15/nodejs
 IMAGEMAGICK_HOME_OSX=/opt/local/Current_v15/ImageMagick
 
 # CFLAGS/CPPFLAGS/CXXFLAGS settings for different platforms. The is most important for Mac
 # where we may need to use non-default SDKs if we're building on a platform newer than the
 # oldest supported
 
-MACOSX_MIN_VERSION=10.15
+MACOSX_MIN_VERSION=12
 ARCHFLAGS="-arch x86_64 -arch arm64"
-SDK_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk
+SDK_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
```

Comment:
SDK_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX14.sdk

Reply:
This is the softlink which was created when i installed this SDK, it always points to the latest.

Reply:
we want latest of 14, so MacOSX14.sdk is what we need